src/main/resources/db_scripts/create_anonymous_database.sql

```diff
@@ -151,7 +151,8 @@ CREATE TABLE anonymous.question_attempts AS
         question_id,
         question_attempt,
         correct,
-        timestamp
+        timestamp,
+        marks
```

Comment:
Again, not important - it won't actually make a difference to the efficiency of the table storage (since it is already after some variable-length columns anyhow), but I would still put marks after correct and before timestamp :/